flit/inifile.py

```diff
@@ -104,10 +104,22 @@ def verify_classifiers(classifiers):
 def read_pkg_ini(path):
     """Read and check the -pkg.ini file with data about the package.
     """
+    cp = _read_pkg_ini(path)
+    return _validate_config(cp, path)
+
+def _read_pkg_ini(path):
     cp = configparser.ConfigParser()
     with path.open() as f:
         cp.read_file(f)
 
+    return cp
+
+
+def _validate_config(cp, path):
+    """
+    validate a config section and return a strip down version fro Metadata
```

Comment:
* 'a config section' -> config (it's not a section, it's the whole thing, but there isn't a good word for that)
* It's not exactly stripped down - the config format includes some extra things under `[metadata]` that aren't strictly part of the distribution metadata. And it returns a dictionary containing several things, only one of which is metadata.